test/e2e/src/common/constants.py

```diff
@@ -101,6 +101,9 @@
 CONTAINER_PERF_RESTART_TIME_EPOCH_QUERY = "Perf | where ObjectName == 'K8SContainer' | where CounterName == 'restartTimeEpoch' |  where TimeGenerated > ago({0}) | count"
 # container log
 CONTAINER_LOG_QUERY = "ContainerLog |  where TimeGenerated > ago({0}) | count"
+CONTAINER_LOG_V2_QUERY = "ContainerLogV2 |  where TimeGenerated > ago({0}) | count"
+CONTAINER_LOG_V2_K8S_METADATA_QUERY = "ContainerLogV2 |  where TimeGenerated > ago({0}) | where KubernetesMetadata != "" | count"
```

Comment:
this is good start and we would need  additional queries to validate kubenetes metadata fields getting ingested or not.